examples/benchmarks/bubblebench/simple-tree/src/appState.ts

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import {
+	type IAppState,
+	type IBubble,
+	type IClient,
+	makeBubble,
+	randomColor,
+} from "@fluid-example/bubblebench-common";
+import type { InsertableTypedNode, TreeView } from "@fluidframework/tree/internal";
+import { type App, type Bubble, Client } from "./schema.js";
+
+export class AppState implements IAppState {
+	readonly localClient: Client;
+
+	constructor(
+		private readonly tree: TreeView<App>,
+		public width: number,
+		public height: number,
+		numBubbles: number,
+	) {
+		this.tree.root.clients.insertAtEnd(
+			this.createInitialClientNode(numBubbles) as unknown as Client,
```

Comment:
This shouldn't need a cast. If it does, something is probably wrong which should be fixed.
```suggestion
			this.createInitialClientNode(numBubbles),
```